### PR TITLE
TGBOT-210_Save_AllBooks_command_as_json_file

### PR DESCRIPTION
**Pull request by MarinaSauchuk**
https://i18nteam.atlassian.net/browse/TGBOT-210

### CODE REVIEW

**Review by lamaconda — APPROVED** [review]
Lgfm

File: src/test/resources/allBooks.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "username": "shelkovich_ilya",
+      "language_code": "ru"
+    },
+    "date": 1682687855,
+    "chat": {
+      "id": 779228565,
+      "type": "private",
+      "first_name": "Илья",
+      "last_name": "Шелкович",
+      "username": "shelkovich_ilya"
```
Comment (travka677) [inline src/test/resources/allBooks.json]: You should deanomize this information

**Review by travka677 — APPROVED** [review]

**Review by QewBye48 — APPROVED** [review]

File: src/test/resources/allBooks.json (hunk 1 above)
Comment (lamaconda) [reply src/test/resources/allBooks.json]: Okay)

**Review by lamaconda — APPROVED** [review]
Lgfm

**Review by QewBye48 — APPROVED** [review]

**Pull request merged**